k8s/node.go

```diff
@@ -0,0 +1,54 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func DeleteNode(k8sClient *kubernetes.Clientset, nodeName string) error {
+	return k8sClient.CoreV1().Nodes().Delete(nodeName, &metav1.DeleteOptions{})
+}
+
+func GetNodeList(k8sClient *kubernetes.Clientset) (*v1.NodeList, error) {
+	return k8sClient.CoreV1().Nodes().List(metav1.ListOptions{})
+}
+
+func CordonUncordon(k8sClient *kubernetes.Clientset, nodeName string, cordoned bool) error {
```

Comment:
I don't think you are using this function anywhere

Reply:
Fixed, used for for worker upgrades.